### DISCUSSION TITLE
Inline color variables don't work

### DISCUSSION THREAD

**Discussion by Celsi** [polybar/polybar#3106] [General]
color1 = #ff0000;
label-mounted = %{T3}  %{T-} %{B${color1}} %mountpoint% %percentage_used%% %{B-}
color1 - don't work, but replace color variable to hex:
label-mounted = %{T3}  %{T-} %{B#ff0000} %mountpoint% %percentage_used%% %{B-}
work all fine

**Comment by Celsi** [comment]
sorry, i'm use 
Linux arch 6.8.1-arch1-1 #1 SMP PREEMPT_DYNAMIC Sat, 16 Mar 2024
i3 v4.23-1
polybar v3.7.1-1

**Comment by patrick96** [comment]
This is a current limitation of the config file. See [this note on the wiki](https://github.com/polybar/polybar/wiki/Configuration#referencing-values):

> **Note:** References can only be used on their own. That means something like key = text ${section.key} text will not work (the reference will not be resolved).
